modules/e2ee/Context.js

```diff
@@ -185,10 +224,20 @@ export class Context {
                     + DIGEST_LENGTH[encodedFrame.type]); // append trailer.
 
                 return crypto.subtle.sign(AUTHENTICATIONTAG_OPTIONS, this._cryptoKeyRing[keyIndex].authenticationKey,
-                    new Uint8Array(newData)).then(authTag => {
+                    new Uint8Array(newData)).then(async authTag => {
```

Comment:
Can we perhaps reorganize this a bit so it uses async / await all the way? It's kinda weird to ready with async / await and promises mixed up.

Reply:
once we're done with the actual code? Until then just keep on piling up things :-)